gallery/plot_datapoints.py

```diff
@@ -0,0 +1,133 @@
+"""
+==============
+Datapoints FAQ
+==============
+
+The :mod:`torchvision.datapoints` namespace was introduced to ``torchvision.transforms.v2``. This example showcases what
```

Comment:
> The :mod:`torchvision.datapoints` namespace was introduced to ``torchvision.transforms.v2``

That's a bit confusing, the `torchvision.datapoints` namespace isn't in the `torchvision.transforms.v2`` namespace